src/utils/verify/verifyCode.ts

```diff
@@ -0,0 +1,3 @@
+export async function verifyCode(): Promise<string> {
+  return Math.floor(100000 + Math.random() * 900000).toString();
```

Comment:
First of all it's better to move it into the verification service. Secondly, it's better to use some cryptography library. You can use the bcrypt, or even `node:crypto` to generate the unique identifier. At least you can use uuid4 algorithm here.

Reply:
I understand your point, but if I'm going to create a verification code using the uuidv4 library (about 20 in length), it would be difficult for a human to enter it. What suggestions do you have? Different methods of creating the code (short for phone, longer for email) or what?

Reply:
If it's email verification code, user will copy it. 
Or you want to implement something like OTP (one time password) - short term verification code that lives about a minute? Like `123 123` ?

Reply:
I did that. The verification code lives a short life (10 minutes in development version). We can change it to a shorter lifetime. I created an endpoint that can repeat sending the code

Reply:
OK. But it's still will be more safe to use `node:crypto` like this https://stackoverflow.com/a/70251173